styles.css

```diff
@@ -6,6 +6,13 @@
   margin: 0;
 }
 
+:root {
+  --inter-font: 'Inter', sans-serif;
+  --style: normal;
+  --flex: flex;
+  --space-between: space-between;
+}
+
 body {
   background-color: #191765;
   background-image: url('Images/Ellipses.png');
```

Comment:
Your design looks so good, but it can be better using Responsive Design techniques, in order to make your design more similar to the Figma design you selected even if you are using a large screen, you can consider adding the next attributes for your `body`:
```
body {
  margin: 0 auto;  // To center the block content
  max-width:  1420px; // To fix the size for large screens
}
```

- [ ] Fix design for large screens

Figma design | Yours (in large screens)

![image](https://user-images.githubusercontent.com/60164239/130188720-9f50e637-756b-4c08-8a1c-db31f3cb8d7e.png)

Reply:
Well solved